src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -263,3 +263,24 @@ def __get__(self, obj, cls):
             return self
         value = obj.__dict__[self.func.__name__] = self.func(obj)
         return value
+
+
+class SessionExpireAtBrowserCloseTool(cherrypy.Tool):
+    def __init__(self):
+        cherrypy.Tool.__init__(self, 'before_finalize',
+                               self._callback)
+
+    def _callback(self):
+        # Shall the cookie expire at browser close?
+        expire_at_browser_close = cherrypy.session.get(
+            'expire_at_browser_close', True)
```

Comment:
Add the `expire_at_browser_close` string as a class attribute with an all caps name.
Then use this class attribute wherever is needed like in `auth.py`